sbt/src/sbt-test/tests/fork-uncaught2/src/test/scala/build/MyFramework.scala

```diff
@@ -0,0 +1,17 @@
+package build
+
+import _root_.sbt.testing._
+
+class MyFramework extends sbt.testing.Framework {
+  def fingerprints = Array(new AnnotatedFingerprint { def isModule = true; def annotationName = "my" })
+  def name = "my"
+  def runner(args: Array[String], remoteArgs: Array[String], testClassLoader: ClassLoader): Runner =
+    new MyRunner(args, remoteArgs, testClassLoader)
+}
+
+class MyRunner(val args: Array[String], val remoteArgs: Array[String],
+               val testClassLoader: ClassLoader) extends sbt.testing.Runner {
+  def tasks(taskDefs: Array[TaskDef]): Array[Task] = throw new Throwable()
```

Comment:
Thought you might be interested: throwing in Task's execute already fails the test task.

But I'm adding it to the test nonetheless.